src/main/resources/config/liquibase/changelog/20190715204745_changelog.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <property name="autoIncrement" value="true"/>
+    <!--
+        Added the entity ExerciseHint.
+    -->
+    <changeSet id="20190715204744-1" author="jhipster">
```

Comment:
please enter your name here @thilo-behnke

Reply:
You are right! I changed it.